### PR TITLE
Update Xamarin bindings for Android SDK

### PR DESCRIPTION
**Pull request by itoys**
Update for 
https://github.com/Azure/azure-notificationhubs-android/releases/tag/v1.0.0-preview1

### CODE REVIEW

File: XPlat/AzureMessaging/Android/samples/NotificationHubsSample/MainActivity.cs [hunk 1]
```diff
@@ -1,192 +1,72 @@
...
+            // TODO: Customize these values to your own
+            const string CONNECTION_STRING = "YOUR-HUB-CONNECTION-STRING";
+            const string HUB_NAME = "YOUR-HUB-NAME";
 
-            #region Check to ensure user has used their own GCM Service, Hub Name, and Hub Secret
-            if (MyRegistrationService.GCM_SENDER_ID == "YOUR-SENDER-ID") {
-                Log ("You must change the SampleGcmBroadcastReceiver.SENDER_IDS to your own Sender ID for the sample to work!");
+            #region Check to ensure user has used their own Connection String and Hub Name
+            if (CONNECTION_STRING == "YOUR-HUB-CONNECTION-STRING")
+            {
```
Comment (mpodwysocki) [inline XPlat/AzureMessaging/Android/samples/NotificationHubsSample/MainActivity.cs:29]: Nit on formatting to use braces on the same line

Comment (itoys) [reply XPlat/AzureMessaging/Android/samples/NotificationHubsSample/MainActivity.cs:29]: it's c# :)

**Review by mpodwysocki — APPROVED** [review]
LGTM

**Comment by itoys** [comment]
@mattleibow @Redth 

Could you please take a look on PR!

And do you know how can I fix build error:

> "/Users/runner/runners/2.168.2/work/1/s/XPlat/AzureMessaging/iOS/samples/NotificationHubsSampleiOS.sln" (Build target) (1:2) ->
> "/Users/runner/runners/2.168.2/work/1/s/XPlat/AzureMessaging/iOS/samples/NotificationHubsSample/NotificationHubsSample.csproj" (default target) (3:6) ->
> (_DetectSigningIdentity target) -> 
>   /Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(696,3): error : Could not find any available provisioning profiles for AzureMessagingSampleiOS on iOS. [/Users/runner/runners/2.168.2/work/1/s/XPlat/AzureMessaging/iOS/samples/NotificationHubsSample/NotificationHubsSample.csproj]

It seems not related to my changes.

Thanks!

**Comment by Redth** [comment]
@mattleibow is looking into the iOS keychain issues in CI :)

**Comment by mattleibow** [comment]
Ah, this is a fork! It doesn't have access to the internal signing keys.

**Review by mattleibow — APPROVED** [review]
LGTM

No breaking changes as well. Nice.

**Pull request merged**